src/bun.js/web_worker.zig

```diff
@@ -280,10 +280,47 @@ pub const WebWorker = struct {
         assert(this.status.load(.acquire) == .start);
         assert(this.vm == null);
 
+        var transform_options = this.parent.transpiler.options.transform_options;
+
+        if (this.execArgv) |exec_argv| parse_new_args: {
+            var new_args: std.ArrayList([]const u8) = try .initCapacity(bun.default_allocator, exec_argv.len);
```

Comment:
maybe we should convert the string impls to `[] const []const u8` so we don't need to hold onto them. `process.execArgv` will cache the result, so this way we don't hold onto them in two places

Reply:
nvm, not doing this unless. parsing the new execArgv will only happen once